gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -639,3 +669,32 @@ func (vm *VMKeeper) QueryFile(ctx sdk.Context, filepath string) (res string, err
 		return res, nil
 	}
 }
+
+// logTelemetry logs the VM processing telemetry
+func logTelemetry(
+	call string,
+	gasUsed int64,
+	cpuCycles int64,
```

Comment:
Would adding variadic attributes here make sense? It would be cool to record the package path and function name to produce aggregates on the computational expense.